### ISSUE TITLE
Crash on save schmeatic

### ISSUE THREAD

**Issue by samwheatley17** [maruohon/litematica#860]
When trying to save a schematic using the "save schematic" button in the area editor, the game goes to the saving world screen and then goes to a black screen, before crashing.

This is the error code:

The game crashed whilst mouseclicked event handler
Error: java.lang.NoSuchMethodError: 'int fi.dy.masa.malilib.gui.GuiTextFieldGeneric.getCursorPosition()'

I also have carpet mod, and tweakeroo installed. I am afraid that this has to do with the stackable shulker boxes, interfering or something, because they have shown some problems aswell. I assume they are trying to do something with your functions and it isnt working correctly.

**Comment by maruohon** [comment]
I assume you have malilib `0.18.4-alpha.1`? Either update Litematica to 0.17.3, or downgrade malilib to 0.18.1.

The `0.18.3` and `0.18.4-alpha.1` versions of malilib were only to fix crashes with the Lunar launcher, and just like I was afraid, making those "fully public" is now starting to cause crashes and issue reports when people don't read the change log or don't update all the mods to the latest versions at the same time. So basically Litematica needs to be 0.17.3+ if you have malilib `0.18.4+`, otherwise Litematica will crash.

**Comment by samwheatley17** [comment]
All versions are on 1.20.6

**Comment by maruohon** [comment]
And you still didn't bother to say which exact mod versions you are using... I think Sakura included those changes to the 1.20.6 versions as well, so you would get the same issue if you have an older Litematica version than malilib.

**Comment by sakura-ryoko** [comment]
Yeah they've been updated to match not long after masa put out the fixes.  Just make sure you are using the latest available 1.20.6 builds.
